### ISSUE TITLE
Un weboai.sqlite par défaut dans data/ ?

### ISSUE THREAD

**Issue by mingarao** [ccahier/weboai#2]
Pour le moment on a `*.sqlite ` dans le gitignore

**Comment by glorieux-f** [comment]
Oui, pour éviter que les gens commitent leur base par inadvertance.
Pour une base par défaut, j'étais perdu avec mes problèmes de droits.
Sur le wiki, tu as vu l'adresse d'un Sitemap testé ?
https://github.com/ccahier/weboai/wiki/Sitemaps

**Comment by glorieux-f** [comment]
Lorsque la base CAHIER sera alimentée, on pourra l’ajouter dans les téléchargements du projet.
